src/app/src/main/kotlin/org/vaccineimpact/api/app/clients/CeleryClient.kt

```diff
@@ -7,11 +7,12 @@ import org.vaccineimpact.api.db.Config
 typealias CeleryTaskResult = Map<String, Map<String, Boolean>>
 typealias CeleryTaskArguments = Array<out Any>
 
-interface TaskQueueClient {
-    fun runDiagnosticReport(group: String, disease: String, touchstone: String): Any
+interface TaskQueueClient
+{
+    fun runDiagnosticReport(group: String, disease: String, touchstone: String, vararg additionalRecipients: String): Any
```

Comment:
why vararg here, will other users have their emails passed in the future?

Reply:
Probably not, I was just making it consistent with the Celery task. But can limit to 1 as that's how it will be in the API in practice